Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -215,13 +222,33 @@ public struct CustomerCenterConfigData {
                     self = .changePlans
                 case "CANCEL":
                     self = .cancel
+                case "CUSTOM_URL":
+                    self = .customUrl
                 default:
                     self = .unknown
                 }
             }
 
         }
 
+        public enum OpenMethod: String {
+
+            case inApp = "IN_APP"
+            case external = "EXTERNAL"
+
+            init?(from rawValue: String?) {
+                switch rawValue {
+                case "IN_APP":
+                    self = .inApp
+                case "EXTERNAL":
+                    self = .external
+                default:
+                    return nil
+                }
+            }
+
+        }
```

Comment:
Hmm right kinda same thinking as you here... It's internal APIs so I guess we can go either way... I would suggest keeping it separate for flexibility for now, and we can always refactor later if needed. For example, we are currently only adding `inApp` and `external` for customer center but we have a `deeplink` option for paywalls... Having said that, I think we could try to refactor everything into a "link handler" to be used by both paywalls and customer center since it's close to the same thing